### PR TITLE
Update translations

### PR DESCRIPTION
**Pull request by TobiGr**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR
update translations from weblate

#### Summary of translated languages

<details><summary>summary</summary>
Translated using Weblate (Korean)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Portuguese (Portugal))

Currently translated at 67.6% (48 of 71 strings)

Translated using Weblate (Portuguese)

Currently translated at 67.6% (48 of 71 strings)

Translated using Weblate (Bengali)

Currently translated at 90.9% (584 of 642 strings)

Translated using Weblate (Portuguese (Portugal))

Currently translated at 99.8% (641 of 642 strings)

Translated using Weblate (Danish)

Currently translated at 91.4% (587 of 642 strings)

Translated using Weblate (Galician)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Estonian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Bengali (Bangladesh))

Currently translated at 64.4% (414 of 642 strings)

Translated using Weblate (Romanian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Portuguese)

Currently translated at 99.8% (641 of 642 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (642 of 642 strings)

Added translation using Weblate (Aymara (Southern))

Added translation using Weblate (Aymará)

Translated using Weblate (Bengali)

Currently translated at 88.9% (571 of 642 strings)

Translated using Weblate (Portuguese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (71 of 71 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 28.1% (20 of 71 strings)

Translated using Weblate (Turkish)

Currently translated at 32.3% (23 of 71 strings)

Translated using Weblate (Galician)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Hindi)

Currently translated at 70.2% (451 of 642 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Persian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Indonesian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Chinese (Traditional, Hong Kong))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Basque)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Japanese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Dutch)

Currently translated at 98.9% (635 of 642 strings)

Translated using Weblate (French)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Japanese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (Japanese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (Sardinian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Polish)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Greek)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (642 of 642 strings)

Translated using Weblate (Japanese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (Japanese)

Currently translated at 99.6% (640 of 642 strings)

Translated using Weblate (German)

Currently translated at 100.0% (642 of 642 strings)

Merge branch 'origin/dev' into Weblate.

Translated using Weblate (Tamil)

Currently translated at 52.6% (337 of 640 strings)

Translated using Weblate (Vietnamese)

Currently translated at 99.5% (637 of 640 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Spanish)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Portuguese)

Currently translated at 60.5% (43 of 71 strings)

Translated using Weblate (Azerbaijani)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Tamil)

Currently translated at 52.5% (336 of 640 strings)

Translated using Weblate (Hebrew)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Portuguese)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Slovak)

Currently translated at 8.4% (6 of 71 strings)

Translated using Weblate (Kurdish (Central))

Currently translated at 98.1% (628 of 640 strings)

Translated using Weblate (Galician)

Currently translated at 99.6% (638 of 640 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Slovak)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (German)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Spanish)

Currently translated at 88.7% (63 of 71 strings)

Translated using Weblate (Hindi)

Currently translated at 4.2% (3 of 71 strings)

Translated using Weblate (Portuguese)

Currently translated at 60.5% (43 of 71 strings)

Translated using Weblate (Hindi)

Currently translated at 68.7% (440 of 640 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Turkish)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Indonesian)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Arabic)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Chinese (Traditional, Hong Kong))

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Italian)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Russian)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Dutch)

Currently translated at 99.3% (636 of 640 strings)

Translated using Weblate (English)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Chinese (Simplified))

Currently translated at 100.0% (71 of 71 strings)

Translated using Weblate (Chinese (Traditional))

Currently translated at 63.3% (45 of 71 strings)

Translated using Weblate (Swedish)

Currently translated at 47.8% (34 of 71 strings)

Translated using Weblate (French)

Currently translated at 90.1% (64 of 71 strings)

Translated using Weblate (Spanish)

Currently translated at 57.7% (41 of 71 strings)

Translated using Weblate (Polish)

Currently translated at 57.7% (41 of 71 strings)

Translated using Weblate (Czech)

Currently translated at 100.0% (71 of 71 strings)

Translated using Weblate (Ukrainian)

Currently translated at 100.0% (71 of 71 strings)

Translated using Weblate (Chinese (Traditional, Hong Kong))

Currently translated at 15.4% (11 of 71 strings)

Translated using Weblate (Portuguese (Portugal))

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Swedish)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Chinese (Traditional, Hong Kong))

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Korean)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (Dutch)

Currently translated at 100.0% (640 of 640 strings)

Translated using Weblate (French)

Currently translated at 100.0% (640 of 640 strings)

Co-authored-by: Agnieszka C <aga_04@o2.pl>
Co-authored-by: AioiLight <info@aioilight.space>
Co-authored-by: Ajeje Brazorf <lmelonimamo@yahoo.it>
Co-authored-by: Alex25820 <alexs25820@gmail.com>
Co-authored-by: Allan Kimmer Jensen <mail@akj.io>
Co-authored-by: Andrew Boonin <catassasin331@gmail.com>
Co-authored-by: Danial Behzadi <dani.behzi@ubuntu.com>
Co-authored-by: Denys Nykula <vegan@libre.net.ua>
Co-authored-by: Eduardo Malaspina <vaio0@swismail.com>
Co-authored-by: Elias Quispe Chura <ilaies_2012@hotmail.com>
Co-authored-by: Eric <hamburger1024@mailbox.org>
Co-authored-by: Error Specialist <errorspecialist02@gmail.com>
Co-authored-by: Fjuro <ifjuro@proton.me>
Co-authored-by: Florin Voicu <florin.bkk@gmail.com>
Co-authored-by: GnuPGを使うべきだ <dieeeazpnnqbpddh@cock.email>
Co-authored-by: Gontzal Manuel Pujana Onaindia <thadahdenyse@gmail.com>
Co-authored-by: Hasan <hasanyildiz0@yaani.com>
Co-authored-by: Hoseok Seo <ddinghoya@gmail.com>
Co-authored-by: Hossain Rizbi <rsajib387@gmail.com>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: Igor Nedoboy <i.nedoboy@mail.ru>
Co-authored-by: Igor Rückert <igorruckert@yahoo.com.br>
Co-authored-by: Ihor Hordiichuk <igor_ck@outlook.com>
Co-authored-by: JS Ahn <freirepublik@gmail.com>
Co-authored-by: JY3 <GeeyunJY3@gmail.com>
Co-authored-by: Jeff Huang <s8321414@gmail.com>
Co-authored-by: Leonardo Brauna <leonardo_brauna@hotmail.com.br>
Co-authored-by: Linerly <linerly@protonmail.com>
Co-authored-by: Maday <royalcoolness7898@gmail.com>
Co-authored-by: Marc Barten <mwbarten@hotmail.com>
Co-authored-by: Marian Hanzel <marulinko@gmail.com>
Co-authored-by: Max Xie <monyxie@gmail.com>
Co-authored-by: MohammedSR Vevo <mohammednajmidin@gmail.com>
Co-authored-by: NTFSynergy <ntfsynergy@gmail.com>
Co-authored-by: Nico Guo <fewis64883@herrain.com>
Co-authored-by: Oymate <dhruboadittya96@gmail.com>
Co-authored-by: Oğuz Ersen <oguz@ersen.moe>
Co-authored-by: Priit Jõerüüt <hwlate@joeruut.com>
Co-authored-by: Ray <ray.cfu@protonmail.com>
Co-authored-by: Rex_sa <rex.sa@pm.me>
Co-authored-by: S3aBreeze <paperwork@evilcorp.ltd>
Co-authored-by: SC <lalocas@protonmail.com>
Co-authored-by: SEENUVASAN T <seenuthiruvpm@gmail.com>
Co-authored-by: TXRdev Archive <lckphanaf9999@gmail.com>
Co-authored-by: Terry Louwers <t.louwers@gmail.com>
Co-authored-by: TiA4f8R <avdivers84@gmail.com>
Co-authored-by: Tom Sawyer <weblate@grymkoll.se>
Co-authored-by: Translator <kvb@tuta.io>
Co-authored-by: Vas R <mrkomododragon1234@gmail.com>
Co-authored-by: Vasilis K <skyhirules@gmail.com>
Co-authored-by: VfBFan <drop0815@posteo.de>
Co-authored-by: Yaron Shahrabani <sh.yaron@gmail.com>
Co-authored-by: atilluF <atilluf@outlook.com>
Co-authored-by: gallegonovato <fran-carro@hotmail.es>
Co-authored-by: gonzalo <misctrashy@gmail.com>
Co-authored-by: komiratsu19273240ad76c354986 <2011945@naver.com>
Co-authored-by: nautilusx <translate@disroot.org>
Co-authored-by: pitachips <hjkim3323@gmail.com>
Co-authored-by: pjammo <adrianoghr@hotmail.it>
Co-authored-by: ssantos <ssantos@web.de>
Co-authored-by: variable virus <variablevirus@gmail.com>
Co-authored-by: weughgh <ahmedhuntingpro@proton.me>
Co-authored-by: zaioti <zaioti@tuta.io>
Co-authored-by: zmni <zmni@outlook.com>
Co-authored-by: 이정희 <daemul72@gmail.com>
Co-authored-by: 정주찬 <ju1801@outlook.com>
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/cs/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/es/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/fr/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/hi/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/pl/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/pt/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/pt_BR/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/pt_PT/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/sk/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/sv/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/tr/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/uk/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/zh_Hans/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/zh_Hant/
Translate-URL: https://hosted.weblate.org/projects/newpipe/metadata/zh_Hant_HK/
Translation: NewPipe/Metadata
</details>

#### APK testing

The APK can be found by going to the "Checks" tab below the title. On the left pane, click on "CI", scroll down to "artifacts" and click "app" to download the zip file which contains the debug APK of this PR.

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

File: app/src/main/res/values-ay/strings.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources></resources>
```
Comment (AudricV) [inline app/src/main/res/values-ay/strings.xml:2]: Was this addition intended?

File: app/src/main/res/values-ayc/strings.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources></resources>
```
Comment (AudricV) [inline app/src/main/res/values-ayc/strings.xml:2]: Same as above.

**Comment by TobiGr** [comment]
That look like new languages. I took a closer look at Weblate: We have 
7 languages at 0% (0 of 721 translated) [ars, ay, ayc, dum, enm, ang, nds, ks, scn]
1 at 1% (1 of 721), [af]
1 at 1% (2 of 721), [kk]
1 at 1% (4 of 721), [si]
1 at 1% (13 of 721) [ace]
I suggest to remove those if no activity occurs until the next RC
